terraform/docker/Dockerfile.terrarium

```diff
@@ -7,29 +7,29 @@ ARG TARGETPLATFORM TARGETOS TARGETARCH TARGETVARIANT BUILDPLATFORM BUILDOS BUILD
 #ENV TARGETARCH_SYNONYM=$([[ "$TARGETARCH" == "amd64" ]] && echo "x86_64" || echo "aarch64")
 #ENV TARGETARCH_SYNONYM_SHORT=$([[ "$TARGETARCH" == "amd64" ]] && echo "x64" || echo "arm64")
 
-ENV TERRAFORM_VERSION=1.4.6 \
-    TERRAFORM_VERSION_DEPRECATED=1.2.9 \
+ENV TERRAFORM_VERSION=1.9.3 \
+    TERRAFORM_VERSION_DEPRECATED=1.4.6 \
     TERRAFORM_CONFIG_INSPECT_VERSION=0.2.0 \
-    TERRAFORM_DOCS_VERSION=v0.16.0 \
-    RUBY_VERSION=3.2.2 \
-    RUBY_VERSION_DEPRECATED=2.7.5 \
-    PACKER_VERSION=1.8.7 \
-    CONSUL_VERSION=1.15.2 \
+    TERRAFORM_DOCS_VERSION=v0.18.0 \
+    RUBY_VERSION=3.3.4 \
+    RUBY_VERSION_DEPRECATED=3.2.2 \
+    PACKER_VERSION=1.11.2 \
+    CONSUL_VERSION=1.19.1 \
     TFENV_VERSION=3.0.0 \
```

Comment:
It might be a good option to use
https://github.com/tofuutils/tenv

It's more versatile and allows us to move forward with additional tooling :-D